app/models.py

```diff
@@ -2878,4 +2878,79 @@ class PhoneMessage(Base, ModelMixin):
     number = orm.relationship(PhoneNumber)
 
 
+class AdminAuditLog(Base):
+    __tablename__ = "admin_audit_log"
+
+    id = sa.Column(sa.Integer, primary_key=True, autoincrement=True)
+    created_at = sa.Column(ArrowType, default=arrow.utcnow, nullable=False)
+    admin_user_id = sa.Column(sa.ForeignKey("users.id"), nullable=False)
+    action = sa.Column(sa.Integer, nullable=False)
+    model = sa.Column(sa.Text, nullable=False)
+    model_id = sa.Column(sa.Integer, nullable=True)
+    data = sa.Column(sa.JSON, nullable=True)
+
+    admin = orm.relationship(User, foreign_keys=[admin_user_id])
+
+    ACTION_CREATE_OBJECT = 1
+    ACTION_UPDATE_OBJECT = 2
+    ACTION_DELETE_OBJECT = 3
+    ACTION_MANUAL_UPGRADE = 4
+    ACTION_EXTEND_TRIAL = 5
+    ACTION_DISABLE_2FA = 6
+    ACTION_LOGGED_AS_USER = 7
+
+    @classmethod
+    def create(cls, **kw):
```

Comment:
This method can be removed if the AdminAuditLog inherits from ModelMixin I think

Reply:
Solved in a previous comment